spec/analytics/ci_spec.rb

```diff
@@ -23,7 +23,13 @@
 
       expect(result).to match({
         "CI" => nil,
-        "key" => "845ac829-2ab3-4bbb-9e24-3529755a6d37"
```

Comment:
I think on line 19 above this should probably change to something like "BUILDKITE env variable not set"